nautobot/core/urls.py

```diff
@@ -60,12 +60,4 @@
         path("", include("django_prometheus.urls")),
     ]
 
-if settings.SOCIAL_AUTH_ENABLED:
-    urlpatterns += [
-        path(
-            "",
-            include("social_django.urls", namespace=settings.SOCIAL_AUTH_URL_NAMESPACE),
```

Comment:
With this being removed, how do the social auth login URLs get registered now?

Reply:
Ah okay. Do they need to be added somewhere else then? This is the only mention of `social_django.urls` that I see in the code base.

Reply:
@glennmatthews Yeah, the SSO stuff has been in a lot of flux in my local environment. We should just go ahead and add it here so it's not resulting in a mixed state once we merge.